lib/cc/yaml/nodes/glob_list.rb

```diff
@@ -3,6 +3,10 @@ module Yaml
     module Nodes
       class GlobList < Sequence
         type Glob
+
+        def allow_child?(child)
```

Comment:
`GlobList` is not the only sequence type that I would expect to be impacted by this. `FetchList` will as well. I also can't think of any circumstance in which we would want `nil` entries in *any* sequence in yaml. Thoughts on making this the default impl?